src/lib/api/RequestFunctions.js

```diff
@@ -0,0 +1,118 @@
+import axios from "axios";
```

Comment:
We can also split this long file into shorter and more readable:
`posts.js`
`users.js`
and so on

Reply:
Done